### PR TITLE
flattened common folder, updated project names

### PR DESCRIPTION
**Pull request by mikelaramie**
removed subfolders from 'common', updated project names to include 'prj-' prefix

### CODE REVIEW

**Comment by mikelaramie** [comment]
Looks like I forgot to update the folder data objects in 2-environments.  I'll clean it up and resubmit.

File: 1-org/projects.tf [hunk 1]
```diff
@@ -150,6 +150,6 @@ module "dns_hub" {
 
   labels = {
     environment      = "prod"
-    application_name = "prj-dns-hub"
+    application_name = "dns-hub"
```
Comment (daniel-cit) [inline 1-org/projects.tf:153]: if the label is fixed, the search for the label needs to be fixed too: 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/e7ef9c22071fd562aba400ff0026cfb60a738a30/3-networks/envs/shared/main.tf#L26

File: 1-org/projects.tf [hunk 2]
```diff
@@ -97,7 +97,7 @@ module "interconnect" {
 
   labels = {
     environment      = "prod"
-    application_name = "prj-interconnect"
+    application_name = "interconnect"
```
Comment (daniel-cit) [inline 1-org/projects.tf:100]: if the label is fixed, the search for the label needs to be fixed too: 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/e7ef9c22071fd562aba400ff0026cfb60a738a30/3-networks/modules/dedicated_interconnect/main.tf#L22

File: 1-org/projects.tf (hunk 1 above)
Comment (mikelaramie) [reply 1-org/projects.tf:153]: Good catch; I'll fix those as well

**Comment by rjerrems** [comment]
Hey @mikelaramie - it looks like there are a bunch of changes showing up on this PR and your other one that aren't a part of the changes you made but are from previous commits to develop?

**Comment by mikelaramie** [comment]
> Hey @mikelaramie - it looks like there are a bunch of changes showing up on this PR and your other one that aren't a part of the changes you made but are from previous commits to develop?

@rjerrems I think I rebase'd incorrectly.  I'll close this one and submit a cleaner PR.

**Pull request closed without merging**